src/test/java/step1/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,48 @@
+package step1;
+
+import org.junit.jupiter.api.Test;
+import step1.domain.StringAddCalculator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringAddCalculatorTest {
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
```

Comment:
다음과 같이 `@NullAndEmptySource`를 활용해서 작성해보시는 것도 좋은 경험이 될 것 같습니다.

```suggestion
    @Test
    @NullAndEmptySource
    public void splitAndSum_null_또는_빈문자(String input) {
```